messages/clientmessages.go

```diff
@@ -56,6 +56,11 @@ type Subscribe struct {
 	Channel string `json:"channel"`
 }
 
+type Unsubscribe struct {
```

Comment:
It would be nice if the struct + it's methods + new function would be grouped next to each other, not scattered around the file